guides/custom-trigger.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: doc
+sublayout: guide
+title: Setting up a Custom Git Trigger
+---
+<p>
+  A Custom Git Trigger is a generic way for any git server to act as a build trigger.<br>
+  It relies solely on SSH keys and its webhook endpoint; everything else is left to the user to implement.
+</p>
+
+<h3>Creating a Trigger</h3>
+
+<p>Creating a Custom Git Trigger is similar to the creation of any other trigger with a few subtle differences:</p>
+<ol>
+  <li>It is not possible for Quay.io to automatically detect the proper robot account to use with the trigger. This must be done manually in the creation process.</li>
+  <li>There are extra steps after the creation of the trigger that must be done in order to use the trigger. These steps are detailed below.</li>
+</ol>
+
+<h3>Post-creation setup</h3>
+
+<p>Once a trigger has been created, there are still 2 steps required before the trigger can be used:</p>
+<ol>
+  <li>Provide read access to the <b>SSH public key</b> generated when creating the trigger.</li>
+  <li>Setup a <b>webhook</b> that POSTs to the Quay.io endpoint to trigger a build.</li>
+</ol>
+<p>The key and the URL are both available at all times by selecting 'View Credentials' from the gear located in the trigger listing.</p>
+
+<h4>SSH public key access</h4>
+
+<p>Depending on the git server setup, there are various ways to install the SSH public key that Quay.io generates for a custom git trigger. For example, <a href="https://git-scm.herokuapp.com/book/en/v2/Git-on-the-Server-Getting-Git-on-a-Server">the git documentation</a> describes a small server setup in which simply adding the key to <code>$HOME/.ssh/authorize_keys</code> would provide access for builders to clone the repository. For any git repository management software that isn't officially supported, there is usually a location to input the key often labeled as "Deploy Keys".</p>
+
+<h4>Webhook</h4>
+
+<p>In order to automatically trigger a build, one must POST a JSON payload to <code>https://%24token:&lt;TOKEN&gt;@quay.io/webhooks/push/trigger/&lt;UUID&gt;</code> with the following format:</p>
```

Comment:
Don't put a semi-real URL here; it might be confusing. We need a more clear way to indicate that they get the URL, including auth, from the dialog